templates/taxi/manufacturer_confirm_delete.html

```diff
@@ -0,0 +1,11 @@
+{% extends "base.html" %}
+
+{% block content %}
+    <h1>You want delete {{ manufacturer }}</h1>
+    <h1>Are you sure to delete this manufacturer?</h1>
```

Comment:
The sentence 'Are you sure to delete this manufacturer?' is slightly awkward. Consider changing it to 'Are you sure you want to delete this manufacturer?' for better clarity.